tests/files/with-typescript-dev-dependencies/package.json

```diff
@@ -1,5 +1,5 @@
 {
   "devDependencies": {
-    "@types/json-schema": "*"
+    "a": "*"
```

Comment:
Now that these tests are actually running, they need a resolved import in order to pass? Otherwise no-extraneous-dependencies ignores them:
https://github.com/import-js/eslint-plugin-import/blob/fd85369fc219c40862d2afcfb11ad053387ba967/src/rules/no-extraneous-dependencies.js#L176